### PR TITLE
Add info for lock cards

### LINKED ISSUE #90: Add tooltips to explain the locks
Add `<Info />` icons with a tooltip to explain what the locks mean:
- Unlockable: Elapsed locks for votes casted on referenda, or track delegation. Unlocking is possible, however, since governance locks can overlap, it doesn't necessarily mean the account's transferable balance will change.

- ~~Locked~~ Unlocking: Funds that will be unlockable after a certain time. These results or either a vote casted on a ref, with conviction, or undelegation on a track.

- Votes: Votes casted on a ref that is still in deciding phase.

### PR DESCRIPTION
**Pull request by Tbaut**
closes #90 
image

---

Submission checklist:

#### Layout

- [x] Change inspected in the mobile web ui
- [x] Change inspected in dark mode

### CODE REVIEW

**Review by wirednkod — DISMISSED** [review]

File: src/components/LocksCard.tsx [hunk 1]
```diff
@@ -131,7 +132,21 @@ export const LocksCard = () => {
...
+                </TooltipTrigger>
+                <TooltipContent>
+                  <p className="max-w-[15rem] text-sm">
+                    Elapsed locks for votes casted on referenda, or tracks
+                    delegation. Since locks can overlap, unlocking doesn't
+                    necessarily mean the transferable balance will change.
+                  </p>
+                </TooltipContent>
+              </Tooltip>
+            </div>
```
[the remark this answers is not in the document]
Comment (Tbaut) [reply src/components/LocksCard.tsx:149]: That's a major issue with this tooltip from shadcn. It's supposedly working on focus as well, but the truth is, it doesn't  work well indeed on Nova. I expect tooltips to be displayed on click. This one on the contrary is even closing on click, so the cursor pointer is super misleading. I'll check if there's a wait to open it on click.

File: src/components/LocksCard.tsx [hunk 2]
```diff
@@ -131,7 +132,21 @@ export const LocksCard = () => {
       {freeLocks.length > 0 && (
         <Card className="relative h-full border-2 p-2 px-4">
           <div className="relative z-10">
-            <Title variant="h4">Unlockable</Title>
+            <div className="flex gap-x-2">
+              <Title variant="h4">Unlockable</Title>
+              <Tooltip>
+                <TooltipTrigger className="cursor-auto">
```
[the remark this answers is not in the document]
Comment (Tbaut) [reply src/components/LocksCard.tsx:138]: Nope, that's on purpose, the pointer is super misleading, you click and it closes the tooltip :roll_eyes:

File: src/components/LocksCard.tsx [hunk 3]
```diff
@@ -205,7 +220,22 @@ export const LocksCard = () => {
       ) : (
         <>
           <Card className="h-full border-2 p-2 px-4">
-            <Title variant="h4">Unlocking</Title>
+            <div className="flex gap-x-2">
+              <Title variant="h4">Unlocking</Title>
+              <Tooltip>
+                <TooltipTrigger className="cursor-auto">
```
[the remark this answers is not in the document]
Comment (Tbaut) [reply src/components/LocksCard.tsx:226]: same

**Comment by Tbaut** [comment]
I'll go for popover instead https://github.com/shadcn-ui/ui/issues/86

**Comment by Tbaut** [comment]
ok that's much better
image

I tested on Nova and it works well

**Review by wirednkod — APPROVED** [review]

**Pull request merged**